source/_docs/ecosystem/caddy.markdown

```diff
@@ -10,12 +10,28 @@ footer: true
 ---
 
 Configure [Caddy Server](https://caddyserver.com/) for use as a reverse proxy to Home Assistant.
-
+## Install
+For the one step installer type:
+```
+curl https://getcaddy.com | bash -s personal
```

Comment:
I know this the caddy recommend approach for one step install, but I'm always super uncomfortable with downloading and passing straight into bash. Maybe throw a warning here for the user?

Reply:
Check out the warning I added.
Do you think it explains everything well?

Reply:
Yeah, I think that works.

Reply:
For warnings you should use:
```
<p class='note warning'>
  TEXT
</p>
```

Reply:
Personally, I found that message a little redundant, maybe shortening it?

something like:
```
It can potentially be dangerous to send a command straight to bash please make sure you trust the source before running that type of commands, to check what this command does head over to [getcaddy.com](https://getcaddy.com/)
```

Reply:
I have changed it
Take a look!!

Reply:
IMO that looks much better 👍 
<https://deploy-preview-6813--home-assistant-docs.netlify.com/docs/ecosystem/caddy/>